src/scripts/Components/Hub.js

```diff
@@ -132,20 +132,24 @@ class Hub extends React.Component {
                 onInstall={this.handleUpdate}
                 onReload={this.handleReload}/>
               {
-                this.props.enableContentHub &&
+                this.props.enableContentHubSearch &&
```

Comment:
```suggestion
                (this.props.enableContentHub || this.props.enableContentHubSearch) &&
```

Reply:
My suggestion is assuming that organizations that are registered on the hub shouldn't always be able to reuse/download content from the hub.